app/models/OkcBlotter/booking.rb

```diff
@@ -1,6 +1,5 @@
 class OkcBlotter::Booking < ApplicationRecord
-  belongs_to :pdf, class_name: 'OkcBlotter::Pdf', foreign_key: 'pdfs_id'
+  belongs_to :pdf, class_name: 'OkcBlotter::Pdf'
   belongs_to :roster, optional: true
-  has_many :offense, dependent: :destroy, foreign_key: 'bookings_id'
-  validates :transient, :inmate_number, :booking_number, :booking_date, presence: true
+  has_many :offenses, class_name: 'OkcBlotter::Offense', dependent: :destroy
```

Comment:
- removed specification of foreign key (didn't need since it follows the convention)
- removed validates for now until we know exactly what's coming through and what's not

Reply:
Offenses had the foreign key of booking_id
Really you never need to specify foreign key here unless it's something weird
Here it's offense_id so we're fine. The weird thing in this case is the class name since it's names paced so that's what we have to specify.